docs/guides/getting_started.md

```diff
@@ -111,8 +111,16 @@ data "openstack_compute_keypair_v2" "mykeypair" {
 
 Use flavor:
 
+> ⚠️ From v0.5.5 the provider supports **only UUID** of a flavor
```

Comment:
i don't think we need this comment since we've never announced that name is allowed